public/index.css

```diff
@@ -165,41 +184,26 @@ h5 {
 		flex-direction: row-reverse;
 		justify-content: flex-end;
 		align-items: baseline;
-		margin-inline-start: calc(-1 * var(--icon-size));
+		margin-inline-start: calc(-1 * var(--icon-size-x));
 	}
 
 	.heading-wrapper > .anchor-link {
-		inset-inline-start: -0.75rem;
-	}
-
-	.heading-wrapper .anchor-icon {
-		position: relative;
-		bottom: var(--icon-baseline, 0);
-	}
-
-	.heading-wrapper.level-h2 .anchor-icon {
-		--icon-baseline: 0.35rem;
-	}
-
-	.heading-wrapper.level-h3 .anchor-icon {
-		--icon-baseline: 0.15rem;
+		margin-inline-end: 0;
 	}
 }
 
-@media (hover: hover) {
-	.heading-wrapper > .anchor-link {
-		opacity: 0;
-	}
+.heading-wrapper > .anchor-link {
+	opacity: 0;
+}
```

Comment:
I don’t think it’s a great idea to hide the anchor links universally. On touch devices where “hover” isn’t really a concept, it makes the anchor links quite hard to discover, except maybe by accident. Once you have accidentally discovered them, you need to poke at some invisible thing on the page, hoping to make the link appear, _then_ click the icon. (Also, with this solution hiding only by opacity it is possible to click on the invisible icon and have the page scroll to the heading, which is a confusing experience in itself.)

I understand the aesthetic concern, which in general you’ve done amazing work on in this PR, but in the end if aesthetic changes hurt UX, I don’t think that’s a worthwhile trade-off. Happy if you want to propose tweaks to how these appear that you think is prettier, but I think we need to stick to hiding them only for `@media (hover: hover)`.

Reply:
I've solved it with 0.5 opacity by default on mobile. 😄